macaroonbakery/tests/test_client.py

```diff
@@ -3,6 +3,7 @@
 import base64
 import datetime
 import json
+from json.decoder import JSONDecodeError
```

Comment:
travis is complaining about this line.

Reply:
I will change it to a ValueError instead which is what the JSONDecodeError is based out of.